src/Apps/Order/Routes/Shipping/Components/SaveAndContinueButton.tsx

```diff
@@ -59,7 +67,7 @@ export const SaveAndContinueButton: React.FC<
       type="submit"
       onClick={onContinueButtonPressed}
       disabled={disableSubmit}
-      loading={shippingContext.state.isPerformingOperation || undefined}
+      loading={isLoading}
```

Comment:
I don't think we should completely remove the check on `isPerformingOperation` - or if we do, we should remove the code as well. Its purpose is specifically about this button's loading/disabled state - [see L36 above](https://github.com/artsy/force/pull/15241/files#diff-73ab7a994382adb29da0fa6958161c467fb1a56fa838cb2ca8ccaa14ff410843R36-R43).